config/config.go

```diff
@@ -554,10 +554,18 @@ var defaultConf = Config{
 		StoreLimit:     0,
 
 		CoprCache: CoprocessorCache{
```

Comment:
usually it's common that users shorten `coprocessor` to `cop`, I think `cop` might be a good name for users.

Reply:
Good idea. We can use this name when we re-introduce it later.